### PR TITLE
feat: add save_charts

### LINKED ISSUE #174: how to save chart to target path?
### 🚀 The feature

how to save chart to target path?

### Motivation, pitch

I found the function of draw chart but no save function found.

### Alternatives

fig.canvas.print_figure

### Additional context

fig.canvas.print_figure

### PR DESCRIPTION
**Pull request by jonbiemond**
Add save_charts argument to init call of PandasAi.
When set to True, a call to `plt.save_fig()` is injected before any calls to `plt.show_plot()` in the run code.
Charts are saved to `exports/charts/<current date>` with a unique name.

- [x] closes #174
- [x] Tests added and passed if fixing a bug or adding a new feature
- [x] All [code checks passed](https://github.com/gventuri/pandas-ai#contributing).
- [x] Integration tested using a modified example
- [x] Integration tested using a Jupyter Notebook

### CODE REVIEW

**Comment by jonbiemond** [comment]
Is this a feature we want to see included in the docs or an example?

**Comment by gventuri** [comment]
@jonbiemond totally, I'd add it in the docs in the paragraph about the plots!

File: README.md [hunk 1]
```diff
@@ -78,17 +82,23 @@ The above code will return the following:
...
 pandas_ai(
     df,
-    "Plot the histogram of countries showing for each the gpd, using different colors for each bar",
+    "Plot the histogram of countries showing for each the gdp, using different colors for each bar",
 )
 ```
 
 ![Chart](images/histogram-chart.png?raw=true)
 
+You can save any charts generated by PandasAI by setting the `save_charts` parameter to `True` in the `PandasAI` constructor. For example, `PandasAI(llm, save_charts=True)`. Charts are saved in `./pandasai/exports/charts` .
```
Comment (jonbiemond) [inline README.md:98]: Saving the charts in the package folder might make them difficult to find for some users, especially when PandasAI is installed into a virtual environment.
Alternatively we could display the save path or/and allow the user to specify a path

Comment (gventuri) [reply README.md:98]: Yeah, printing the newly generated path is a great idea, let's go for it!

**Pull request merged**